src/plugins/gpunetio/gpunetio_backend.cpp

```diff
@@ -404,6 +405,8 @@ nixlDocaEngine::nixlDocaInitNotif (const std::string &remote_agent,
     delete notif->recv_mmap;
     delete notif->recv_barr;
 
+    delete notif;
```

Comment:
seems aldo need to free `notif->send_addr`

Reply:
also recv_addr